action/src/utils.ts

```diff
@@ -96,8 +112,17 @@ export class Utils {
         core.addPath(cliDir);
     }
 
-    public static getCliUrl(major: string, version: string, fileName: string): string {
+    public static getCliUrl(major: string, version: string, fileName: string, releasesRepo: string): string {
         let architecture: string = 'frogbot-' + Utils.getArchitecture();
+        if (releasesRepo) {
+            let platformUrl: string = process.env.JF_URL ?? '';
+            if (!platformUrl) {
+                throw new Error('failed while downloading Frogbot from Artifactory, JF_URL must be set');
```

Comment:
Unlike Go, errors in Typescript are capitalized
```suggestion
                throw new Error('Failed while downloading Frogbot from Artifactory, JF_URL environment variable must be set');
```

Reply:
Done